<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompanyDialog</class>
 <widget class="QDialog" name="CompanyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Транспортная компания</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Тип транспорта:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="typeCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="schedulesLabel">
     <property name="text">
      <string>График перевозок:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="schedulesTable">
     <column>
      <property name="text">
       <string>Отправление</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Прибытие</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата отправления</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата прибытия</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Цена</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Места</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addScheduleButton">
       <property name="text">
        <string>Добавить рейс</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editScheduleButton">
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteScheduleButton">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

